### PR TITLE
Initial Feed Integration Development Documentation

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
In Progress

## Description
Very barebones right now. Lots of it needs clarification/elaboration. We need to provide concrete examples and images. Look at the documentation for [fetch-incidents](https://github.com/demisto/content-docs/blob/master/docs/fetching-incidents.md) to see how this could be made comprehensive.

### CODE REVIEW

File: docs/feeds.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Feed Integrations
+
+## How to Implement a Feed Integration
+
+**Requirements**:  
+- Having the field `feed: true` in the script part of their yaml.
```
Comment (glicht) [inline docs/feeds.md]: Add Demisto Server 5.5 as a requirement

**Comment by glicht** [comment]
@avidan-H are you working on this? 

Also, please merge from master. There is a new layout to the site.

**Comment by avidan-H** [comment]
@glicht I'll flesh it out now.

**Comment by avidan-H** [comment]
@glicht awaiting review.

**Pull request merged**